src/server/server.js

```diff
@@ -0,0 +1,10 @@
+const http = require('http');
+const router = require('./router.js');
+require('dotenv').config()
```

Comment:
You're already using env2 in another file: dotenv is the same thing so isn't not necessary to have both.